test/e2e/apimachinery/flowcontrol.go

```diff
@@ -265,22 +265,24 @@ func getPriorityLevelConcurrency(f *framework.Framework, priorityLevelName strin
 
 // createFlowSchema creates a flow schema referring to a particular priority
 // level and matching the username provided.
-func createFlowSchema(f *framework.Framework, flowSchemaName string, matchingPrecedence int32, priorityLevelName string, matchingUsername string) (*flowcontrol.FlowSchema, func()) {
+func createFlowSchema(f *framework.Framework, flowSchemaName string, matchingPrecedence int32, priorityLevelName string, matchingUsernames ...string) (*flowcontrol.FlowSchema, func()) {
 	var subjects []flowcontrol.Subject
-	if matchingUsername == "*" {
+	if len(matchingUsernames) == 1 && matchingUsernames[0] == "*" {
```

Comment:
you're right, I don't think we need the `*` clause anymore

Reply:
presumably this is to let another test keep working the same?

Reply:
no, the only test that uses `*` is the fairness drown-out test

Reply:
removed "*" codes